### PR TITLE
use cli for messages

### PR DESCRIPTION
**Pull request by pat-s**
Most chatty messages have been replaced by `cli` functions.

I could not test all options of every step in the wizard because I faced #102. 

Would be great if you could test it on your side with a fake repo again. 

After approval, I'll edit the README. 

~~(This PR includes a styling of the pkg using `styler::style_pkg()`.)~~

This cluttered the diff too much.

### CODE REVIEW

**Comment by krlmlr** [comment]
Deploy preview for *eloquent-wescoff-5685d5* ready!

Built with commit a46cb77bb7c9b02a46243497802c555369a80835

https://deploy-preview-103--eloquent-wescoff-5685d5.netlify.com

**Comment by pat-s** [comment]
@krlmlr 

#102 issue starts in `travis_set_pat()`.

First I thought https://github.com/ropenscilabs/travis/pull/103/files#diff-3df713d1b07460a6eb8151e5cbc07dfe being the problem but it was not.
Before wasting more time on this, could you have look? I guess you might have an immediate idea what is going on here.
Works with `master` but maybe you can also double-check this on your side since we are dealing with third-party stuff.

### Traceback

```r
traceback()
20: stop(http_condition(x, "error", task = task, call = call))
19: httr::stop_for_status(auth_travis, "authenticate with travis") at travis-auth.R#20
18: `_f`(gh_token = gh_token)
17: withVisible(`_f`(gh_token = gh_token))
16: auth_travis() at travis-auth.R#43
15: `_f`(repo = repo)
14: withVisible(`_f`(repo = repo))
13: travis_token(repo) at travis-pat.R#23
12: paste("token", token) at travis-api.R#10
11: request(headers = c(..., .headers))
10: httr::add_headers(Authorization = paste("token", token)) at travis-api.R#10
9: named(list(...))
8: handle_url(handle, url, ...)
7: httr::GET(travis(url), httr::user_agent("ropenscilabs/travis"), 
       accept, httr::add_headers(Authorization = paste("token", 
           token)), ...) at travis-api.R#10
6: TRAVIS_GET(url, ..., httr::add_headers(`Travis-API-Version` = 3), 
       token = token) at travis-api.R#18
5: TRAVIS_GET3(sprintf("/repo/%s/env_vars", encode_slug(repo)), 
       token = token) at travis-vars.R#17
4: travis_get_vars(repo = repo, token = token) at travis-vars.R#58
3: travis_get_var_id(name = name, repo = repo, token = token) at travis-vars.R#103
2: travis_set_var("GITHUB_PAT", pat, public = FALSE, repo = repo, 
       token = token, quiet = quiet) at travis-pat.R#23
1: travis::travis_set_pat()
```

**Comment by krlmlr** [comment]
Will do later. The setup already looks so much better with the boxes and {travis} in the current `master`.

File: R/github-key.R [hunk 1]
```diff
@@ -44,11 +44,12 @@ github_add_key <- function(pubkey, title = "travis+tic",
   # add public key to repo deploy keys on GitHub
   ret <- add_key(key_data, repo, gh_token, quiet)
 
-  message(
-    "Successfully added public deploy key '", title, "' to GitHub for ", repo, ". ",
-    "You should receive a confirmation e-mail from GitHub. ",
-    "Delete the key in the repository's settings to revoke access for that key or when you no longer need it."
-  )
+  cli::cat_bullet(bullet = "tick", bullet_col = "green",
+    sprintf("Successfully added added public deploy key ", title, "' to GitHub for ", repo, ". "))
```
Comment (krlmlr) [inline R/github-key.R]:
```suggestion
    sprintf("Successfully added public deploy key ", title, "' to GitHub for ", repo, ". "))
```

**Comment by krlmlr** [comment]
Thanks!

**Comment by pat-s** [comment]
@krlmlr Did you have a look at the error? In the current state `travis_set_pat()` is broken.

**Pull request merged**